src/Icicle/Storage/Dictionary/Toml.hs

```diff
@@ -150,10 +153,7 @@ checkDefs d defs
  where
   go (a, q)
    = do  (checked, _)  <- check' d q
-         let inlined    = P.sourceInline d checked
-         blanded       <- hoistEither . mapLeft DictionaryErrorTransform $ P.sourceDesugarQT inlined
-         (checked', _) <- check' d blanded
-         pure $ DictionaryEntry a (VirtualDefinition (Virtual checked'))
+         pure $ DictionaryEntry a (VirtualDefinition (Virtual checked))
```

Comment:
it's silly, but the main reason is so we keep more or less the original, "nicer" version before we make it ugly